README.rst

```diff
@@ -96,13 +96,13 @@ care of the dependencies: ``scikit-learn``, ``Rpy2``, and
 Installation
 ~~~~~~~~~~~~
 
-Now ``pip`` installable!
+Install the stable PyPI version with ``pip``
 
 .. code:: bash
 
     $ pip install pyglmnet
 
-Manual installation instructions below:
+For the bleeding edge development version:
```

Comment:
I think we should give a one line installer for the bleeding edge:

```sh
$ pip install https://api.github.com/repos/glm-tools/pyglmnet/zipball/master
```

Reply:
And what is below should go into a `Contributing.md` file

Reply:
agree, i'd like to test this before making the change though and it leads to scope creep. capturing in a separate issue and resolving here

Reply:
see #320